inst/dust/daedalus.cpp

```diff
@@ -307,7 +305,7 @@ class daedalus_ode {
 
     // RESPONSE AND VACCINATION CLASSES
     std::vector<size_t> idx_hosp =
-        daedalus::helpers::get_state_idx({iH + 1}, n_strata, N_VAX_STRATA);
+        daedalus::helpers::get_state_idx({iHd + 1}, n_strata, N_VAX_STRATA);
```

Comment:
I expect you want to include `Hr` here as well, as otherwise only `Hd` are counted as being hospitalised in responses.

```suggestion
        daedalus::helpers::get_state_idx({iHd + 1, iHr + 1}, n_strata, N_VAX_STRATA);
```